02 - Community Expansions/02.01 Plugins by Category/Backup plugins.md

```diff
@@ -20,4 +20,10 @@ Plugins to backup your notes.
 
 %% Add links to other 02.02 - Category notes %%
 
-#placeholder/notes
```

Comment:
Deleted & added line: why?

Reply:
There was no end-of-line character in the original file.

This original meant that the `%% Hub footer: Please don't edit anything below this line %%` comment ran right up against the end of the content.

I made the script first effectively:

1. add the missing end-of-line character at the end of the file, and then
2. add the footer.

Git shows this up as an edit on the last line of the file...